src/main/java/xyz/nikitacartes/easyauth/commands/LoginCommand.java

```diff
@@ -70,11 +75,18 @@ private static int login(ServerCommandSource source, String pass) throws Command
             else if (maxLoginTries == 1) {
                 player.networkHandler.disconnect(TranslationHelper.getWrongPassword());
                 return;
+            } else if (playerCache.getLoginTries() == maxLoginTries - 1 && maxLoginTries != -1) {
+                player.networkHandler.disconnect(TranslationHelper.getLoginTriesExceeded());
+                playerCache.incrementLoginTries();
+
+                // Reset their login try counter after the amount of seconds specified in the config.
+                RESET_LOGIN_THREAD.schedule(playerCache::resetLoginTries, config.experimental.resetLoginAttemptsTime, TimeUnit.SECONDS);
```

Comment:
you're probably right actually.

Reply:
I'll probably add some logic in AuthEventHandler for the player trying to join and check the time difference. The only thing that'd kinda suck is daylight saving time but quite a bit of other mojang code is also not dst-safe

Reply:
Aight I did the above. It works nicely and actually should persist across restarts (in case someone decides to use it as a 1 week tempban or smth). Turns out I overcomplicate things. A lot. And no, this is not the first time i've done that lol